test/integration/targets/yarn/tasks/run.yml

```diff
@@ -43,8 +43,8 @@
       register: yarn_version
 
     - name: 'Create dummy package.json'
-      template:
-        src: package.j2
+      copy:
+        src: templates/package.j2
```

Comment:
It's better to put static files into files folder instead of templates and use normal file extension instead of `.j2`, since it's not a template.

Reply:
@verkaufer if you don't mind filing a follow-up PR to fix that, I'd appreciate it. Thanks :)